src/modules/Completelist.js

```diff
@@ -0,0 +1,28 @@
+import LocalStorage from './localStorage.js';
+
+class Completelist {
+  static CompleteToDoList() {
+    const checkbox = document
+      .querySelector('.to-do-list')
+      .querySelectorAll('.checkbox');
+
+    checkbox.forEach((elemnt) => {
+      elemnt.addEventListener('change', (e) => {
+        LocalStorage.taskComp(e.target.parentElement.parentElement.id);
+      });
+    });
+  }
+
+  static clearCompleted() {
+    const clearButton = document.querySelector('.clear');
+    clearButton.addEventListener('click', () => {
+      let todolists = LocalStorage.getToDoLists();
+      todolists = todolists.filter((elemnt) => elemnt.comp !== true);
+      localStorage.setItem('todolists', JSON.stringify(todolists));
+      window.location.reload();
+      LocalStorage.resetIndex();
+    });
+  }
+}
+
```

Comment:
- Hey, when users click the checkboxes, they don't get updated from true to false in the local storage and the clear all completed doesn't seem to clear any completed task. please, double-check this to make sure your app meets this part of the project requirement as much as you can.